AppKit/CPWindow/CPWindow.j

```diff
@@ -3844,6 +4069,46 @@ var interpolate = function(fromValue, toValue, progress)
 
 @end
 
+@implementation CPWindow (TrackingArea)
+
+- (void)_addToViewsWithTrackingAreas:(CPView)aView
```

Comment:
Change this name to `_addTrackingAreaView`. Cocoa method names follow the pattern \<verb>\<object type>:\<object>